distributed/asyncio.py

```diff
@@ -117,17 +108,15 @@ class AioClient(Client):
     """
     _Future = AioFuture
 
-    def __init__(self, *args, loop=None, start=True, set_as_default=False,
-                 **kwargs):
```

Comment:
I agree that explicitly passing a loop object smells like a bad idiom. If a current thread's loop is not adequate, then perhaps the loop setup is wrong.